src/test/ui/mismatched_types/issue-38371.stderr

```diff
@@ -1,46 +1,35 @@
 error[E0308]: mismatched types
-  --> $DIR/issue-38371.rs:4:8
+  --> $DIR/issue-38371.rs:8:8
    |
-LL | fn foo(&foo: Foo) {
-   |        ^^^^-----
-   |        |     |
-   |        |     expected due to this
+LL | fn foo(&_a: Foo) {}
+   |        ^^^  --- expected due to this
+   |        |
    |        expected struct `Foo`, found reference
-   |        help: did you mean `foo`: `&Foo`
    |
    = note: expected struct `Foo`
            found reference `&_`
+help: to take parameter by ref, move `&` to the type
```

Comment:
we have the binding ident, we could use that to name this?

also `by ref` => `by reference` maybe?

Reply:
Hm, so like "to take parameter \`x\`"? 🤔

Reply:
yep!

Reply:
Done 👍🏻